### DISCUSSION TITLE
Allow me to skip Issues that are marked with priority Important in my project

### DISCUSSION THREAD

**Discussion by paddywills** [actions/stale#1172] [General]
As I assign every Issue to a project, I want to skip any Issues that are set to Important priority.
Could this be possible?

**Comment by panjf2000** [comment]
I was looking for the same thing, some issues are indeed inactive but also vital and should be kept around. For example, some issues of feature requests that are already accepted and I don't have time to implement them for the moment. These issues shouldn't be considered stale and closed by the bot automatically.

What I have in mind is that `actions/stale` adds a new option to allow users to specify a label and skip the issues tagged with that label.
